### PR TITLE
Feature/tabs

### PR DESCRIPTION
**Pull request by mahsa-ameri**
- Added Tabs component
- Update the nav style + create a new style file for the Tabs component to match with the Figma
- Added Tabs to mahsa-qa page

### CODE REVIEW

**Review by MikeJalow — APPROVED** [review]

File: component-library/component-lib/src/lib/tabs/tabs.component.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+};
+export interface ITabConfig {
+  id?: string,
+  title: string,
+  value?: string
+}
+@Component({
+  selector: 'lib-tabs',
+  templateUrl: './tabs.component.html',
+  styleUrls: ['./tabs.component.css']
```
Comment (xingruoxicic) [inline component-library/component-lib/src/lib/tabs/tabs.component.ts:19]:
```suggestion
```
Suggest remove this and the empty css file.

Comment (mahsa-ameri) [reply component-library/component-lib/src/lib/tabs/tabs.component.ts:19]: Good catch! 
Done, thanks! :)

File: component-library/component-lib/src/lib/tabs/tabs.component.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  previousID = '';
+
+  @Input() config: ITabNavConfig = {
+    id: '',
+    size: 'large'
+  }
+
+  @Output() click: EventEmitter<any> = new EventEmitter();
+
+  constructor(private cd: ChangeDetectorRef) { }
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/tabs/tabs.component.ts:32]: Remove

File: component-library/component-lib/src/lib/tabs/tabs.component.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  id?: string,
+  title: string,
+  value?: string
+}
+@Component({
+  selector: 'lib-tabs',
+  templateUrl: './tabs.component.html',
+  styleUrls: ['./tabs.component.css']
+})
+export class TabsComponent implements OnInit, AfterViewInit {
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/tabs/tabs.component.ts:21]: Refactor to use the approach I sent you.

File: component-library/component-lib/src/lib/tabs/tabs.component.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  templateUrl: './tabs.component.html'
+})
+export class TabsComponent implements OnInit {
+
+  @Input() config: ITabNavConfig = {
+    id: '',
+    size: 'large'
+  }
+  
+  constructor() { }
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/tabs/tabs.component.ts:26]: Remove if unused

**Review by kristopher-houston — APPROVED** [review]

**Pull request merged**